boards/openhwgroup/cv32a6_genesys_2/cv32a6_genesys_2_defconfig

```diff
@@ -0,0 +1,14 @@
+# Copyright 2024 CISPA Helmholtz Center for Information Security gGmbH
+# SPDX-License-Identifier: Apache-2.0
+CONFIG_INCLUDE_RESET_VECTOR=y
+CONFIG_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_UART_NS16550_ACCESS_WORD_ONLY=y
+CONFIG_CONSOLE_HANDLER=y
+CONFIG_XIP=n
+CONFIG_INIT_STACKS=y
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=25000000
+# RNG
+CONFIG_TIMER_RANDOM_GENERATOR=y
+CONFIG_TEST_RANDOM_GENERATOR=y
```

Comment:
why are these here?

Reply:
At some point in the past, not even a basic hello-world example would build without these. This appears to be fixed, so I removed them.